clevertap-core/build.gradle

```diff
@@ -28,6 +28,8 @@ android {
     kotlinOptions {
         jvmTarget = JavaVersion.VERSION_1_8
     }
+    namespace 'com.clevertap.android.sdk'
+    testNamespace 'com.clevertap.demo'
```

Comment:
not sure why is this change needed

Reply:
https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#namespace-dsl
Namespace is now needed in the gradle file and not the manifest file

Reply:
The testNamespace is picked up from the AndroidManifest of androidTest of the core module